frontend/src/views/Shares/ShareView.js

```diff
@@ -686,6 +712,66 @@ const ShareView = () => {
                   </Card>
                 </Grid>
               </Grid>
```

Comment:
If I am picky and push this a bit by adding super long reasons, if the users do not start new lines (which works fine) they will lose part of their messages in the frontend

<img width="1507" alt="image" src="https://github.com/awslabs/aws-dataall/assets/71252798/470ad10e-9432-437c-b661-d2812c70e50b">

Reply:
Good call out - added to the frontend `sx={{ wordBreak: "break-word" }}` and tested to ensure words can be broken and wrapped to new lines for both reject and request Purpose